src/components/MachineTranslationCallout.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import {
+  Callout,
+  CreateIssueButton,
+  Icon,
+  Trans,
+  useTranslation,
+  Link,
+} from '@newrelic/gatsby-theme-newrelic';
+
+const MachineTranslationCallout = ({ englishHref }) => {
+  const { t } = useTranslation();
+  return (
+    <Callout
+      variant="important"
+      title=" "
+      css={css`
+        margin: 2em 0;
+        padding: 0.5em;
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          align-items: center;
+          > * {
+            margin: 0 1em;
+          }
+          @media screen and (max-width: 1240px) {
+            flex-direction: column;
+            align-items: flex-start;
+            > * {
+              margin: 1em 0;
+            }
+            > :last-child {
+              align-self: flex-end;
+            }
+          }
+        `}
+      >
+        <Icon name="globe" size="2em" />
+        <div>
+          <h4>{t('machineTranslation.calloutHeading')}</h4>
+          <Trans i18nKey="machineTranslation.calloutBody" parent="p">
+            In the event of any inconsistency between the English version and
+            the translated version, the{' '}
+            <Link to={englishHref} shouldAutoLocalize={false}>
+              English version
+            </Link>
+            shall prevail. Please visit{' '}
```

Comment:
"shall prevail" conjures images of pointy hats and magic wands for me. Would something like "In the event of any inconsistency between the English version and the translated version, the English version will take priority." This is just my opinion, feel free to disregard.